pyproject.toml

```diff
@@ -30,7 +30,7 @@ keywords = [
 dynamic = ['version']
 dependencies = [
     "attrs>=22.2.0",
-    "botorch>=0.8.1",
+    "botorch>=0.9.3",
```

Comment:
For this reason, we need to wait until we dropped python 3.8. This comment is here in order to block merging.